test/blockchain/log-processors/crowdsourcers.js

```diff
@@ -26,12 +27,12 @@ const getDisputesFromCrowdsourcer = (db, params, callback) => {
 const getCrowdsourcerAndMarket  = (db, params, callback) => {
   getCrowdsourcer(db, params, (err, crowdsourcer) => {
     if (err) return callback(err);
-    db("markets").first(["marketId", "reportingRoundsCompleted"])
-      .where({ marketId: crowdsourcer.marketId })
-      .asCallback((err, market) => {
+
+    getMarketsWithReportingState(db).where({ "markets.marketId": crowdsourcer.marketId })
+      .asCallback((err, markets) => {
         if (err) return callback(err);
 
-        callback(null, { market, crowdsourcer });
```

Comment:
You can still call first() empty if you like and it'll maintain the previously selected columns, but either works